src/renderToString.js

```diff
@@ -40,7 +40,7 @@ function renderNode(children, node) {
  * @param {Object} data
  * @return {String}
  */
-export default function renderToString(data) {
+export default function renderToString(data, pretty = false) {
```

Comment:
Lets make the second argument and options object so it's easier to add things in the future.